pyvisfile/xdmf/__init__.py

```diff
@@ -310,7 +310,7 @@ def from_dtype(dtype: np.dtype) -> "DataItemNumberType":
             return DataItemNumberType.Int
         elif dtype.type in (np.uint8, np.uint16, np.uint32, np.uint64, np.uint):
             return DataItemNumberType.UInt
-        elif dtype.type in (np.float16, np.float32, np.float64, np.float128):
```

Comment:
* Please link the issues you found in the code (thanks!)
* `np.float128` and `np.longdouble` are still distinct types. We should support whatever is available.

Reply:
> * Please link the issues you found in the code (thanks!)
Not 100% sure what you mean here - should I add a link to this PR here?

> * `np.float128` and `np.longdouble` are still distinct types. We should support whatever is available.
I've made the dtype determination more generic, so this should be ok now.

Reply:
Yeah, lgtm.